src/components/markdown/references/references.jsx

```diff
@@ -0,0 +1,214 @@
+import React, { useContext } from 'react'
+import PropTypes, { array } from 'prop-types'
```

Comment:
Could remove the `array` import, since it's not used.

```suggestion
import PropTypes from 'prop-types'
```